mautic/mautic
# Page tracking submissions in landing page

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               |no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | N/A




#### Description:

Added new column "Results" to landing pages table, it shows number of leads who submitted a form in the specific landing page.
This item is an item that directs to an other page that shows a list of submissions in that specific landing page, similar to form submissions page.

All items in  the table are sortable, filterable and exportable, and each item has a reference to form.

### Before
![landing-pages](https://user-images.githubusercontent.com/50916237/108218045-8e9faf80-7134-11eb-9c88-bb7fdb2abe5c.png)


### After
![results-page](https://user-images.githubusercontent.com/50916237/108217843-59935d00-7134-11eb-9cae-a8f59bfc47cb.png)




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create a form
3. Create a landing page with form
4. Submit form in landing page
5. Check that landing pages table has result columns
6. Click number of results
7. Check the results table with your submission




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9693"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
